e2e_tests/tests/fixtures/unmanaged/checkpointing.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+
+import random
+import time
+
+from determined.experimental import core_v2
+
+
+def main():
+    name = f"unmanaged-checkpoints-{int(time.time())}"
```

Comment:
that is a good idea, switched to do this. Is there a way to lookup experiment by external_experiment_id besides listing every experiment?

There is still a difference between the example that is relevant. The example doesn't have a storage path passed while the fixture does, and not having any checkpoint storage specified will make the storageID of the checkpoint null.

Reply:
we have a `PUT` by experiment id: https://github.com/determined-ai/determined/blob/main/proto/src/determined/api/v1/api.proto#L371 
I'd suggest we add a `GET` for the same URL.

Reply:
maybe just as a field on GetExperiment would work too? 

Created this
https://hpe-aiatscale.atlassian.net/browse/DET-10095

Reply:
it's a `GET` request with the parameter coming from the URL. I don't see how one endpoint could support both parameters in this case?

we have model registry APIs which treat `model_name` parameter magically either as an id or as a name, but I do not like it because it's a time bomb.

Reply:
oh yeah my bad, I suppose I was thinking more about adding a field on GetExperiments

I was thinking of how we have name on ```GetGroups``` which should also only ever return one result. I guess this still annoying since then the client has to do its own assertion that exactly one item is returned

I updated the ticket to be more general on the solution